scout/test/pageflow/page_load_status.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+"""
+Tests pages and their respective URL status codes
+"""
+
+import sys
+from django.test import TestCase
+
+
+class UrlStatusTest(TestCase):
+
+    def clientUrlStatus(self, urlsuffix=''):
```

Comment:
I'm merging this, but highly recommend that in a later commit you set these two methods apart from the tests with an underscore:

def _clientUrlStatus